app/classifier/index.cjsx

```diff
@@ -5,6 +5,7 @@ ClassificationSummary = require './classification-summary'
 {Link} = require 'react-router'
 tasks = require './tasks'
 drawingTools = require './drawing-tools'
+{session} = require '../lib/session'
```

Comment:
I think we'd like to be able to distinguish between these scenarios, each on a shared computer:

  - Person A opens browser, classifies a bit, closes browser, leaves computer. Person B sits down at computer 10 minutes later & does exactly the same thing. (Goal: we guess it's 2 different people.)
  - Person A opens browser, classifies, pauses to get a cup of coffee, comes back to computer, refreshes browser window, continues classifying. (Goal: we guess it's the same person.)
  - Person A classifies not-logged-in until prompted, then logs in, continues classifying, then closes browser. Person B sits down immediately, opens browser and does exactly the same thing. (Goal: we guess B's not-logged-in classifications don't belong to A.)

Hope this is helpful/realistic/posted in the right place?